packages/monaco/package.json

```diff
@@ -6,16 +6,16 @@
     "@theia/core": "^1.4.0",
     "@theia/editor": "^1.4.0",
     "@theia/filesystem": "^1.4.0",
-    "@theia/monaco-editor-core": "^0.19.3",
+    "@theia/monaco-editor-core": "^0.20.0",
     "@theia/markers": "^1.4.0",
     "@theia/outline-view": "^1.4.0",
     "@theia/workspace": "^1.4.0",
     "deepmerge": "2.0.1",
     "fast-plist": "^0.1.2",
     "idb": "^4.0.5",
-    "jsonc-parser": "^2.0.2",
+    "jsonc-parser": "^2.2.0",
```

Comment:
please check with `yarn why jsonc-parser` that we don't have duplicates and bump up all places

Reply:
Done, squashed to https://github.com/eclipse-theia/theia/pull/8010/commits/6703948146759691397a4552f0859dfc2d7479f1